setup.py

```diff
@@ -76,7 +76,7 @@
         "backports.cached_property ; python_version<'3.8'",
         "click>=8.0.0",
         "eth-account==0.5.7",
-        "ethpm-types>=0.1.0b7",
+        "ethpm-types>=0.1.0b7,<0.1.1",
```

Comment:
Add a note here to update this pin when `ethpm-types` is released post ape v0.2.0 minor release